modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1132,6 +1322,87 @@ private Folder _getFolder(NotificationQueueEntry notificationQueueEntry)
 				notificationQueueEntry.getNotificationQueueEntryId()));
 	}
 
+	private Set<String> _getTermNames(
+			long objectDefinitionId, Set<String> selectedTermCategories)
+		throws Exception {
+
+		Set<String> termNames = new HashSet<>();
+
+		MockLiferayResourceRequest mockLiferayResourceRequest =
+			new MockLiferayResourceRequest();
+
+		mockLiferayResourceRequest.addParameter(
+			"objectDefinitionId", String.valueOf(objectDefinitionId));
+
+		MockLiferayResourceResponse mockLiferayResourceResponse =
+			new MockLiferayResourceResponse();
+
+		_mvcResourceCommand.serveResource(
+			mockLiferayResourceRequest, mockLiferayResourceResponse);
+
+		ByteArrayOutputStream byteArrayOutputStream =
+			(ByteArrayOutputStream)
+				mockLiferayResourceResponse.getPortletOutputStream();
+
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray(
+			byteArrayOutputStream.toString());
+
+		for (int i = 0; i < jsonArray.length(); i++) {
+			JSONObject jsonObject = jsonArray.getJSONObject(i);
```

Comment:
rename `jsonObject` to `ftlTermCategoryJSONObject` because it is an item of `ftlTermCategoriesJSONArray`